components/collection/unlockable/utils/useUnlockableTag.ts

```diff
@@ -0,0 +1,68 @@
+import { getDropDetails } from '@/components/drops/useDrops'
+
+const useUnlockableTagTexts = ({
+  isMintedOut,
+  isFree,
```

Comment:
to me, creating a non exported composable seems a little pointless
computing the labels directly in `useUnlockableTag` seems more straight forward

Reply:
move to useUnlockableTag